build-image/first-boot.bash

```diff
@@ -64,7 +64,8 @@ if [[ -z "${username+x}" ]] || ! id $userdef &> /dev/null || id "$username" &> /
 else
   usermod -l "$username" "$userdef"
   usermod -m -d "/home/$username" "$username"
-  groupmod -n "$username" "$userdef"
+  groupmod -n "openhab" "$userdef"
+  usermod --append --groups "$username" "$userdef"
```

Comment:
What exactly is `$userdef` for, I can see that it is selected based on if it is a pi or not but what for?

Reply:
IDK for sure didn't write that... it's set some lines up.
Bottom line is that the user of that name already exists, has an environment, sudo rights etc.